src/app-router.js

```diff
@@ -28,7 +28,11 @@ export class AppRouter extends Router {
   reset() {
     super.reset();
     this.maxInstructionCount = 10;
-    this._queue = [];
+    if (!this._queue) {
+      this._queue = [];
```

Comment:
Well, if it actually being reset, it will be assigned. The constructor also runs "reset" to set everything to the initial state. That's when the queue would get initially set. It's a little ugly, but it works.